addons/web/static/src/webclient/actions/action_service.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+  <t t-name="web.BlankComponent" owl="1">
+    <ControlPanel display="{disableDropdown: true}" t-if="props.withControlPanel and !env.isSmall">
+        <t t-set-slot="layout-buttons">
+            <button class="btn btn-primary invisible"> empty </button>
```

Comment:
class="invisible"? I have no idea what the purpose of this component is, but if it's never displayed, there may be no need to make it translatable